android/build.gradle

```diff
@@ -45,7 +54,7 @@ try {
         }
         if (!file(sdkDir).isDirectory()) throw new Exception()
         // Couldn't find this documented anywhere; see https://android.googlesource.com/platform/tools/build/+/6d7fd0d2eff092abf1aaf44d03756b24570b390c/gradle/src/main/groovy/com/android/build/gradle/internal/Sdk.groovy#187
-        System.properties["android.home"] = property("sdk.dir")
+        System.properties["android.home"] = sdkDir
```

Comment:
How does this change work?

Reply:
The `sdkDir` variable is setup just a few lines above. It might be set from `property("sdk.dir")`, or it might be set from `getenv("ANDROID_HOME")`. Since we've gone to that trouble, we might as well use it here.